db/seeds.rb

```diff
@@ -64,6 +64,7 @@
 num_users = 10 * SEEDS_MULTIPLIER
 
 users_in_random_order = seeder.create_if_none(User, num_users) do
+
```

Comment:
Can you please remove this empty line? Rubocop complains about it:

![image](https://user-images.githubusercontent.com/47985/124863344-1ea99d00-dfe1-11eb-8dfc-1ae4a5529fda.png)

```suggestion
```

Reply:
Sorry, I removed it and force pushed.